koswat/strategies/strategy_reinforcement_type_costs.py

```diff
@@ -8,17 +8,17 @@
 @dataclass
 class StrategyReinforcementTypeCosts:
     reinforcement_type: type[ReinforcementProfileProtocol]
-    base_costs: float = 0.0
     base_costs_with_surtax: float = 0.0
     infrastructure_costs: float = 0.0
```

Comment:
Is this still needed? I'd expect, as well as with `base_costs` that it gets removed.

Reply:
It is still used for the summary, accessed via `StrategyLocationInput.get_infrastructure_costs`.